package.json

```diff
@@ -198,6 +198,7 @@
     "qs": "^6.9.4",
     "react-cosmos": "^5.6.6",
     "react-test-renderer": "^18.0.0",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
You should include it in dependencies and make sure it is included in the non-test code, because ResizeObserver is not supported on Safari on iOS 12.5, which has >0.2% market share.

We define valid browsers in our package.json under browserslist, and we support all browsers with >0.2% market share (except Opera Mini and dead browsers). You can check whether an API is available under this subset of browsers using https://caniuse.com/?search=ResizeObserver (feel free to ask for a demo).